testing/kokoro/system_tests.sh

```diff
@@ -21,16 +21,22 @@ set -x
 go version
 date
 
-# Re-organize files
-export GOPATH=$PWD/gopath
-target=$GOPATH/src/github.com/GoogleCloudPlatform
-mkdir -p $target
-mv github/golang-samples $target
-cd $target/golang-samples
+cd github/golang-samples
 
 export GO111MODULE=on # Always use modules.
 export GOPROXY=https://proxy.golang.org
 
+# Fail if a dependency was added without the necessary go.mod/go.sum change
+# being part of the commit.
+# Do this before reserving a project since this doens't need a project.
+for i in `find . -name go.mod`; do
```

Comment:
Are there multiple go.mod? If not you can just use the inner block

Reply:
Yes. Some samples need to be standalone modules.